### PR TITLE
Issue #90: Adding support for windows 10 SDK

### PR DESCRIPTION
**Pull request by T-Jin**
I ran into some issues with building glui on windows 10. Made a quick fix.
I later saw the exact same issue was previously reported in issue #90, but no PR yet. So here it is.

### CODE REVIEW

File: include/GL/glui.h [hunk 1]
```diff
@@ -38,8 +38,13 @@
 #include <OpenGL/GL.h>
 #include <OpenGL/GLU.h>
 #else
-#include <GL/gl.h>
-#include <GL/glu.h>
+# ifdef _WIN32
+#  define NOMINMAX
+#  include <Windows.h>
```
Comment (nigels-com) [inline include/GL/glui.h]: `#include <windows.h>` I think is more conventional.

Have you seen Windows.h in common usage?

Comment (T-Jin) [reply include/GL/glui.h]: Letter case doesn't matter on Windows. I do agree I see lower case ones more often in older c/c++ code. Win10 sdk named the file `Windows.h` though.

The file itself is definitely needed if you are doing any Windows specific stuff. A more likely scenario is that by including some Windows specific header, like `winsock2.h` for networking, `windows.h` would be included for you by those headers. So you don't need to explicitly include it most of the time.

**Comment by nigels-com** [comment]
I think by convention it's better for the client code to include the Windows headers, along with decisions about NOMINMAX and so on.  Declining this change.

**Comment by nigels-com** [comment]
Ah there is more to this story.

There was a change in 2017 to remove the GLUT dependency from the GLUI header.
A side-effect of that is that now Windows builds break because GLUT is/was including `windows.h`

https://github.com/libglui/glui/pull/76

**Comment by nigels-com** [comment]
Modern GLUT is now going to great lengths to avoid the `windows.h` dependency...
https://github.com/markkilgard/glut/blob/master/include/GL/glut.h

Should GLUI also do the same, at least for (more) compatibility with the previous release?

**Comment by T-Jin** [comment]
> Ah there is more to this story.
> 
> There was a change in 2017 to remove the GLUT dependency from the GLUI header.
> A side-effect of that is that now Windows builds break because GLUT is/was including `windows.h`
> 
> #76

I think it was the right move to remove GLUT dependency from GLUI header. I noticed that GLUI Windows build with FreeGLUT also breaks because FreeGLUT is/was including `windows.h`.

Now knowing that, the original changes I made in this PR is not a good solution, with all that windows namespace pollution.

**Comment by T-Jin** [comment]
> Modern GLUT is now going to great lengths to avoid the `windows.h` dependency...
> https://github.com/markkilgard/glut/blob/master/include/GL/glut.h
> 
> Should GLUI also do the same, at least for (more) compatibility with the previous release?

Wow, those guys really went through all that effort just to get those macro defined. I would be inclined to do the same for GLUI.

**Comment by nigels-com** [comment]
Out of interest.

Does this also work for previous Windows and SDK versions, or just the current one?

**Comment by nigels-com** [comment]
Hello @T-Jin this PR is still active for GLUI, but it looks like you're using the branch for other things.  Do you want to restart a review, or happy for this to be closed?

**Comment by T-Jin** [comment]
Ah, apology for the noise, I should have created the PR on a dedicated branch. I don't have the time to follow it through, so let me close this for now

**Pull request closed without merging**